configure.ac

```diff
@@ -513,7 +513,8 @@ AS_IF([test "$install_completion" != no],
      [if test "$prefix" = "NONE"; then
         completions="`pkg-config --variable=completionsdir bash-completion`"
       else
-        completions="`pkg-config --define-variable=prefix=$prefix --variable=completionsdir bash-completion`"
+        # bash-completion >= 2.10 uses datadir instead of prefix
+        completions="`pkg-config --define-variable=prefix=$prefix --define-variable=datadir=$prefix --variable=completionsdir bash-completion`"
```

Comment:
`$datadir` is normally `$prefix/share`, no?

Reply:
Oh, yeah, right, this won't work.